test/api_access_test.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class ScopesTest < Minitest::Test
+  def test_write_is_the_same_access_as_read_write_on_the_same_resource
+    full_scopes = ShopifyAPI::ApiAccess.new(%w(read_orders write_orders))
+    expected_scopes = ShopifyAPI::ApiAccess.new(%w(write_orders))
+    assert_equal expected_scopes, full_scopes
```

Comment:
How would you feel about having more descriptive variable names in these tests?

Like:

```suggestion
    read_write_orders = ShopifyAPI::ApiAccess.new(%w(read_orders write_orders))
    write_orders = ShopifyAPI::ApiAccess.new(%w(write_orders))
    assert_equal write_orders, read_write_orders
```

I know it's explained in the test name, but it feels like it'd be easier to read what the test is actually asserting this way.